README.md

```diff
@@ -98,6 +98,7 @@ export default App;
 
 ## Changelog
 
+- **[1.1.0](https://github.com/Dwolla/react-drop-ins/releases/tag/v1.1.0)** Updated Dwolla Web to v2.2.0, and added new component implementations. See GitHub Releases for more information.
```

Comment:
The Releases URL will 404 for now, but will populate when the PR is merged and tagged. :)